# param/p2d_ndt_node.param.yaml
---

/**:
  ros__parameters:
    observation_sub:
      history_depth: 10
    map_sub:
      history_depth: 10
    pose_pub:
      history_depth: 10
    publish_tf: true
    predict_pose_threshold:
      translation: 0.5
      rotation: 0.1
    localizer:
      map:
        capacity: 55000
        min_point:
          x: -130.0
          y: -130.0
          z: -3.0
        max_point:
          x: 130.0
          y: 130.0
          z: 3.0
        voxel_size:
          x: 2.0
          y: 2.0
          z: 2.0
      scan:
        capacity: 55000
      optimization:
        outlier_ratio: 0.55
      optimizer:
        max_iterations: 50
        score_tolerance: 0.001
        parameter_tolerance: 0.001
        gradient_tolerance: 0.001
        line_search:
          step_max: 0.12
          step_min: 0.0001
      guess_time_tolerance_ms: 5

